bindings/python/src/torrent_handle.cpp

```diff
@@ -460,7 +460,7 @@ void bind_torrent_handle()
 #ifndef TORRENT_DISABLE_DHT
         .def("force_dht_announce", _(&torrent_handle::force_dht_announce))
 #endif
-        .def("scrape_tracker", _(&torrent_handle::scrape_tracker))
+        .def("scrape_tracker", _(&torrent_handle::scrape_tracker), arg("index") = -1)
```

Comment:
I was under the impression that you had to provide names for any argument preceding the one you wanted to give a default as well. But I suppose you tested this, that it applies to the last parameter, and not the first